azure/storage/stress/handler/cloud_table_handler.rb

```diff
@@ -11,81 +11,331 @@ module Handler
+          temp = 
+            if tableList.inlcude?(tableName)
+              LoggingAspect::logger.info("Table #{tableName} exists when creating")
+              nil
+            else
+              LoggingAspect::logger.info("Create table: #{tableName}")
+              tableClient.create_table(tableName, reqOptions)
+            end
+        else
+          LoggingAspect::logger.info("Create table: #{tableName}")
+          temp = tableClient.create_table(tableName, reqOptions)
+        end
+
         # ==== Construct Return Value ==== #
+        result = XSS::AutoGenerated::CloudTableService::Create_result.new
+        result.success = temp.nil?
+        result
       end
 
-
-
       def exists(requestInfo, accountInfo)
         tableClient = XSS::Converter::TableConverter.getTableService(self, accountInfo)
+        
         # ==== Construct Parameters ==== #
+        tableName = requestInfo.tableName
+        reqOptions = XSS::Converter::CoreConverter.getRequestOptions(requestInfo.thriftRequestOptions)
+        reqOptions.merge! XSS::Converter::CoreConverter::getOperationContextOptions(requestInfo.thriftOperationContext)
+
         # ==== Operation ==== #
+        LoggingAspect::logger.info("Get table (exists?): #{tableName}")
+        exist = false;
+        begin
+          tableClient.get_table(tableName, reqOptions)
+          exist = true
+        rescue
+        end
+
         # ==== Construct Return Value ==== #
+        result = XSS::AutoGenerated::CloudTableService::Exists_result.new
+        result.success = exist
+        result
       end
 
-
-
       def deleteTable(deleteIfExists, requestInfo, accountInfo)
         tableClient = XSS::Converter::TableConverter.getTableService(self, accountInfo)
+        
         # ==== Construct Parameters ==== #
+        tableName = requestInfo.tableName
+        reqOptions = XSS::Converter::CoreConverter.getRequestOptions(requestInfo.thriftRequestOptions)
+        reqOptions.merge! XSS::Converter::CoreConverter::getOperationContextOptions(requestInfo.thriftOperationContext)
+
         # ==== Operation ==== #
+        LoggingAspect::logger.info("Delete table: #{tableName}")
+        temp = tableClient.delete_table(tableName, reqOptions)
+
         # ==== Construct Return Value ==== #
+        result = XSS::AutoGenerated::CloudTableService::DeleteTable_result.new
+        result.success = temp
+        result
       end
 
-
-
       def setPermissions(reqInfo, accountInfo, permissions)
         tableClient = XSS::Converter::TableConverter.getTableService(self, accountInfo)
+
         # ==== Construct Parameters ==== #
+        tableName = requestInfo.tableName
+        reqOptions = XSS::Converter::CoreConverter.getRequestOptions(requestInfo.thriftRequestOptions)
+        reqOptions.merge! XSS::Converter::CoreConverter::getOperationContextOptions(requestInfo.thriftOperationContext)
+        signedIdentifiers = []
+        permissions.each do |permission|
```

Comment:
Can use `XSS::Converter::CoreConverter.getSignedIdentifiersFromThriftSharedAccessPolicies` to convert